content/040_modules/1_setup.md

```diff
@@ -18,7 +18,7 @@ Cloud credentials provide privileges to programmatically interact with your Amaz
 
 - Login to the Rafay Console with the credentials provided and click on Infrastructure
 
-- Select "Cloud Credentials", Click on "New Credential" and provide a unique name "aws-workshop-x" where x aligns with your Rafay user.
+- Select "Cloud Credentials", Click on "New Credential" and provide a unique name "aws-cloudcredential-xx" where xx aligns with your Rafay user.
 
 ![Create Cloud Credential](/040_modules/img/part1/cloud_credential_create.png)
```

Comment:
For all static images can you please put these under static/images? It's easier to keep track of all static assets this way. Example: ![Create Cloud Credential](/images/c9create.png)